### PR TITLE
8166026: Refactor java/lang shell tests to java

### PR DESCRIPTION
**Pull request by frkator**
Refactor `test/jdk/java/lang/SecurityManager/modules/CustomSecurityManager.sh` as java test.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8166026](https://bugs.openjdk.java.net/browse/JDK-8166026): Refactor  java/lang shell tests to java

### Reviewers
 * [Sean Mullan](https://openjdk.java.net/census#mullan) (@seanjmullan - **Reviewer**) ⚠️ Review applies to 6968fee1f396d6ca29432b73084fbe456f90c41f

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/1579/head:pull/1579`
`$ git checkout pull/1579`

### CODE REVIEW

**Review by seanjmullan — APPROVED** [review]

File: test/jdk/java/lang/SecurityManager/modules/CustomSecurityManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2015, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (seanjmullan) [inline test/jdk/java/lang/SecurityManager/modules/CustomSecurityManagerTest.java:2]: You are missing a comma after "2020".

**Comment by frkator** [comment]
@iignatev could you please sponsor this? Thank you!

**Pull request closed without merging**